utils.js

```diff
@@ -748,12 +823,30 @@ function Signer (password, isTestnet, networks, SLIP44, pubTypes) {
   this.accountIndex = 0
 }
 function TrezorSigner (password, isTestnet, networks, SLIP44, pubTypes) {
+  try{
+    if(!trezorInitialized){
+      console.log('Initializing trezor')  
+      TrezorConnect.init({
+        // connectSrc: 'https://localhost:8088/', // un-comment for testing on your localhost connect server
```

Comment:
this should be configurable from pali or consuming wallet incase user wants a non-trusted backend running their own